pkg/storage/unified/client.go

```diff
@@ -17,6 +17,7 @@ import (
 	"github.com/grafana/grafana/pkg/infra/tracing"
 	"github.com/grafana/grafana/pkg/services/apiserver/options"
 	"github.com/grafana/grafana/pkg/services/authn/grpcutils"
+	"github.com/grafana/grafana/pkg/services/authz"
```

Comment:
Is this able to be 
https://github.com/grafana/authlib/blob/main/authz/client.go#L104